ZigbeeNet/CC/IHardwareChannel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace ZigbeeNet.CC
```

Comment:
Is the interface just temporary in this Namespace? as you mentioned in the XML comment it should be implemented by all Hardware implementation, but CC is TI specific

Reply:
Yes